docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+---
+layout: vNext
+title:  "Setting up and Running a Performance Benchmark on an existing network"
+categories: 
+permalink: /vNext/fabric-tutorial/tutorials-fabric-existing/
+order: 
+---
+## Table of Contents
+{:.no_toc}
+
+- TOC
+{:toc}
+
+## Overview
+
+This tutorial takes you through performance testing a smart contract on a pre-existing Fabric network using Caliper. 
+
+This tutorial follows from the test-network [fabric tutorial](https://hlf.readthedocs.io/en/latest/test_network.html). You need to have installed and instantiated the fabcar smart contract (part two of the Fabric tutorial).
+
+
+## Step 1 - Install and Bind Caliper
+
+Globally install caliper CLI using the following terminal command:
+
+`npm install -g --only=prod @hyperledger/caliper-cli@0.4.0`
+
+Bind the SDK using the following terminal command:
+
+`caliper bind --caliper-bind-sut fabric:2.2.0 --caliper-bind-args=-g`
```

Comment:
should we use fabric:2.2 rather than 2.2.0 ?

Reply:
Also the node-sdk has a slight problem which means we can't use any of the 2.2 sdks at the moment because fabric-common forgets to declare it's dependency on long so loading fabric-network fails because long is missing. This I think only happens because the node-sdk modules are loaded into global and node.js require doesn't really do loading modules from global with explicitly putting it on the path. I am assuming it is put into global so caliper-cli will pick it up as it's installed globally as well ?

Reply:
Requiring global packages (e.g., caliper-core from the workload module) can be enabled with the following env var: https://github.com/hyperledger/caliper/blob/master/packages/caliper-publish/caliper.Dockerfile#L35-L37

Reply:
A fix I submitted has gone into the node-sdk, however I've changed the SUT bindings for 2.2 so we don't explicitly load protos and common, which means that we avoid the bug in the node-sdk